tfhe/src/integer/server_key/radix_parallel/tests_unsigned/test_scalar_comparison.rs

```diff
@@ -736,6 +768,10 @@ mod no_coverage {
         // but with param 3_3 we actually encrypt more that 128bits
         PARAM_MESSAGE_4_CARRY_4_KS_PBS_GAUSSIAN_2M64
     });
+
+    create_parametrized_test!(integer_comparisons_for_empty_blocks {
+        PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64,
+    });
```

Comment:
```suggestion
    create_parametrized_test!(integer_comparisons_for_empty_blocks)
```

Reply:
This would require MultiBitPBSParameter support, which isn't done in the other functions. Should I instead add more parameter sets?

Reply:
Hum, then you can do

```rust
    create_parametrized_test!(integer_comparisons_for_empty_blocks {
        PARAM_MESSAGE_1_CARRY_1_KS_PBS_GAUSSIAN_2M64,
        PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64,
        PARAM_MESSAGE_3_CARRY_3_KS_PBS_GAUSSIAN_2M64,
        PARAM_MESSAGE_4_CARRY_4_KS_PBS_GAUSSIAN_2M64
    });
```